src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -218,14 +220,24 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
     config_dp_endpoint = get_config_dp_endpoint(cmd, location)
 
     # Retrieving Helm chart OCI Artifact location
-    registry_path = os.getenv('HELMREGISTRY') if os.getenv('HELMREGISTRY') else utils.get_helm_registry(cmd, config_dp_endpoint, dp_endpoint_dogfood, release_train_dogfood)
+    if container_registry_repository:
+        if disable_auto_upgrade is False:
+            auto_upgrade_error = 'Disable auto upgrade when using custom repository'
+            telemetry.set_exception(exception=auto_upgrade_error, fault_type=consts.Custom_Repository_Disable_Auto_Upgrade_Fault_Type, summary=auto_upgrade_error)
+            raise ValidationError(auto_upgrade_error, recommendation='use --disable_auto_upgrade')
+
+        registry_path = "{}/azurearck8s/azure-arc-k8sagents:1.3.8".format(container_registry_repository)
```

Comment:
Why is 1.3.8 hardcoded here?
 If it requires to be hardcoded, please add it in consts file.

Reply:
Also, are we always assuming that the customer's using pcr will always have the 1.3.8 version present? What if in future we release a new agent version and a new pcr customer uploads the new agent version into his pcr instead of 1.3.8

Reply:
it should not be hardcoded. in the connect command, we should take in --agent-version and use that to specify here. as long as the customer has ensured that they pushed the chart to <repository>/azurearck8s/azure-arc-k8sagents:<agent_version>, we should be good. if the chart is not found at that location based on the inputs provided by the customer, we throw an error

Reply:
ack